synapse/handlers/account.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, hs: "HomeServer"):
 
     async def get_account_statuses(
         self,
-        user_ids: List[str],
+        user_ids: Sequence[str],
```

Comment:
I think we try not to do `Sequence[str]` since a `str` is also a `Sequence[str]`?

Reply:
Oh true. Context:

- I wanted to annotate the `user_ids` on the request body as immutable where possible.
- But I also wanted to use Pydantic's `conlist()` to enforce a minimum length---regrettably there doesn't seem to be a `contuple()`.

I'll drop the first bullet (which will be no worse than today).